packages/api-explorer/src/Doc.jsx

```diff
@@ -418,6 +419,7 @@ Doc.propTypes = {
   ),
   language: PropTypes.string.isRequired,
   lazy: PropTypes.bool,
+  loginUrl: PropTypes.string,
```

Comment:
Can you add a `.isRequired` to this or set `https://dash.readme.com/login` as the default?

Reply:
Hmm yeah I was waffling on how to handle this since technically it won't break anything if it's omitted. I'll add a default in the Log component.

Reply:
Updated: https://github.com/readmeio/api-explorer/pull/683/commits/b3fe21d11acf42ffac17950bc9428dd4323b2946